### PR TITLE
General improvements

### PR DESCRIPTION
**Pull request by SolalPirelli**
Three main areas:

* Merge the concepts of "contest judge" and "contest organizers" (I don't remember there being a specific "contest judge" in previous editions anyway)

* Clarify unauthorized behavior and its consequences

* Minor improvements, typo fixes, etc.

### CODE REVIEW

File: Rules.md [hunk 1]
```diff
@@ -1,36 +1,34 @@
...
-2. ## Conditions of participation
+2. ## Participation Conditions
 
   2.1 **Eligibility**
 
-    3. Participants must currently live, study or work in Switzerland, and be at least 14 years old.
+    2. Participants must currently live, study or work in Switzerland, and be at least 14 years old.
 
-    4. Participants take part in the contest in teams of up to 3 people. Single-person teams are allowed.
+    3. Participants must take part in the contest in teams of 1, 2 or 3 people.
```
Comment (tharvik) [inline Rules.md]: Do we still have somewhere that we can merge teams if needed?

File: Rules.md [hunk 2]
```diff
@@ -58,168 +56,151 @@ The Contest environment should allow first and foremost the dissemination of kno
...
 
+    46. Teams may not:  
+    
+    - sleep, behave rudely, or interfere in any way with the contest organization;
+    - try to gain an unfair advantage over their competitors;
+    - bring food or drinks into the computer labs;
+    - take problem statements, printouts and draft papers out of the computer labs;
+    - share information with other teams;
+    - write or say anything related to the contest in any place where it might be seen or heard by other teams;
+    - attempt to overload, crash or exploit the judging system, or to obtain a larger amount of computational resources than is allowed.
```
Comment (tharvik) [inline Rules.md]: We can explicit the fact that we disallow multi-threading, it may not be obvious for the reader.

**Comment by tharvik** [comment]
Otherwise, LGTM.

File: Rules.md (hunk 1 above)
Comment (SolalPirelli) [inline Rules.md]: We never had that in the rules, and AFAIK it was never enforced against teams' will.

However, this raises a good point: we should make it explicit that participants can join teams (and not just leave them) before the registration deadline. I'll add that.

File: Rules.md (hunk 2 above)
Comment (SolalPirelli) [inline Rules.md]: IMHO we should have a full list of unauthorized behavior somewhere. I know multithreading and filesystem access are forbidden, but what else?

**Comment by SolalPirelli** [comment]
Merging since nobody else has commented on it.

**Pull request merged**